README.md

```diff
@@ -1 +1,153 @@
-# math_magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [Math-Magician ](#math-magician-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Math-Magician <a name="about-project"></a>
+
+**Math-Magician** is a web based single page calculator that allows users to perform simple mathematical operations.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project was built using these technologies.
+
+- JavaScript
+- Webpack
+- HTML
+- CSS
+- React
+- Git/Github
+- Node.js
+  
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Lets users perform mathematical operation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> In Progress...
+<!-- - [Live Link]() -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+This project requires the following:
+
+- Node.js
+- Webpack
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git 
+  cd Math-React
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+    npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+    npm test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Michael Kithinji**
+
+- GitHub: [@githubhandle](https://github.com/MICHAELKITH)
+- Twitter: [@twitterhandle](https://twitter.com/DevMichael11)
+- LinkedIn: [LinkedIn](linkedin.com/in/dev-mike-5a09ab167)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] @MICHAELKITH, Your project is completed however please kindly add Future Features section else it will be invalidated.
<img width="622" alt="image" src="https://user-images.githubusercontent.com/52934128/226597941-fffd112a-4cde-4d63-983e-9b2016088ae0.png">